pokete.py

```diff
@@ -527,12 +527,16 @@ class Station(se.Square):
     choosen = None
     obs = []
 
-    def __init__(self, associate, additionals, width, height, char="#",
-                 w_next="", a_next="", s_next="", d_next="", state="solid"):
+    def __init__(self, associate, additionals, width, height, desc,
+                 char="#", w_next="", a_next="", s_next="", d_next="",
+                 state="solid", label_fn=None):
+        self.desc = desc
         self.org_char = char
+        self.label_fn = label_fn
         self.associates = [associate] + [ob_maps[i] for i in additionals]
         self.color = ""
-        super().__init__(char, width, height)
+        self.name = self.associates[0].pretty_name
+        super().__init__(char, width, height, state="float")
```

Comment:
`state="solid"` is not assigned in `__init__`, so it could either be removed, assigned to `self` or passed to `super().__init__` as an argument.

Reply:
Fixed, this was an relic.